dotcom-rendering/src/components/Badge.tsx

```diff
@@ -65,7 +64,7 @@ export const Badge = ({
 						: frontsSectionBadgeSizingStyles,
 				]}
 				src={imageSrc}
-				alt=""
+				alt={isInLabsSection ? 'Labs sponsor logo' : ''}
```

Comment:
Thanks @deedeeh for picking up the missing alt attribute. Adding something (anything!) here will improve the user experience, at it will be announced when the user selects the link.

So, as it is currently defined, if the badge appears in the Labs section, the screen reader will announce "Labs sponsor logo, link". This is a lot better than nothing, but it still might be difficult for users to understand.

Otherwise, if the alt attribute is empty, the behaviour is undefined, but it might announce part of the destination URL, which is very difficult to understand.

At present, the Badge component doesn't receive enough information to provide to the user a meaningful indication of where they will be directed when they click the link.

Can the Badge component receive a new prop called `sponsorName` or similar, which is derived from the [`Branding` interface](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/branding.ts#L18-L24)?

Reply:
That is what I wanted to do to have the name of the sponsor but what I found the badge has just `imageSrc` and `href` properties. We might receive more data in badge but I will need to check. I am not sure how complex it might be but I will let you know if I need help. 

I will timebox it and if there is complexity I will merge this one and have that change in a separate PR.